package.json

```diff
@@ -65,7 +65,7 @@
   },
   "homepage": "https://github.com/DataDog/dd-trace-js#readme",
   "engines": {
-    "node": ">=16"
+    "node": ">=14"
```

Comment:
Was this added on accident?

Reply:
I'm currently testing aerospike with node 14 in this PR, but once it gets approved i'll remove the test for it from the CI and revert the package.json. After merging the pr, node 14 will get tested in the v3 release branch

Reply:
Then the PR should be a draft, otherwise it could be merged by accident.